### PR TITLE
adding SORN and Agency

### PR DESCRIPTION
**Pull request by RyanSibley**
I added two definitions and edited the entries that were here from the beginning (I think they were hold-overs from FOIAOnline).

### CODE REVIEW

File: js/util/glossary/terms.js [hunk 1]
```diff
@@ -3,11 +3,15 @@ const terms = [
...
     term: 'agency component',
     definition: `
-      A thing that receives a FOIA request. This is a catch-all term used by
-      OIP to refer to departments, bureaus, and offices that receive FOIA
+      This is a catch-all term used to refer to departments, bureaus, and offices that receive FOIA
       requests.`,
   },
 
+   {
+    term: 'Agecny',
```
Comment (adborden) [inline js/util/glossary/terms.js]: Typo.

**Review by adborden — APPROVED** [review]

**Comment by adborden** [comment]
@RyanSibley I can get this merged for you. I notice you're merging into the `rs-more-glossary-adds` branch instead of the `develop` branch. Do you want the changes on `rs-more-glossary-adds` merged too?

**Comment by RyanSibley** [comment]
I just saw your comment @adborden -- yeah, I got confused and set up the branch wrong. I'm sorry if it caused any problems.

**Comment by adborden** [comment]
No worries just making sure I got everything. Merging this in a few minutes once the tests are pass.

**Pull request merged**